Sources/SwiftFormatConfiguration/RuleRegistry+Generated.swift

```diff
@@ -12,8 +12,8 @@
 
 // This file is automatically generated with generate-pipeline. Do Not Edit!
 
-enum RuleRegistry {
-  static let rules: [String: Bool] = [
+public enum RuleRegistry {
```

Comment:
We should use an `@_spi` here; we don't want this table to be public API to anyone who uses `SwiftFormat`.

`@_spi(Internal)` would be fine, I think.

Reply:
Thank you for the review!

Sure, this seems reasonable — let me clean this up a bit later today.

I want to also add the pluralization of the warning message, and see if I can get source location to work _easily_. I think that `<unknown>` in CLI output (when it's the only line) does not make it clear that it's supposed to be the source location, and it's just empty.